typing_extensions/src_py2/typing_extensions.py

```diff
@@ -0,0 +1,113 @@
+import abc
+import collections
+import typing
+from typing import (
+    ClassVar, Type,
+    Counter, DefaultDict, Deque,
+    NewType, overload, Text, TYPE_CHECKING,
+)
+
+# Please keep __all__ alphabetized within each category.
+__all__ = [
+    # Super-special typing primitives.
+    'ClassVar',
+    'Type',
+
+    # Concrete collection types.
+    'Collection',
+    'ContextManager',
+    'Counter',
+    'Deque',
+    'DefaultDict',
+
+    # One-off things.
+    'NewType',
+    'overload',
+    'Text',
+    'TYPE_CHECKING',
```

Comment:
OK, let us not include ``GenericMeta`` for now. People who want to use it can just import it from ``typing`` (it just does not work with ``import *``).